core/blockchain.go

```diff
@@ -485,7 +487,10 @@ func (bc *BlockChain) dispatchTxUnindexer() {
 // - updating the acceptor tip index
 func (bc *BlockChain) writeBlockAcceptedIndices(b *types.Block) error {
 	batch := bc.db.NewBatch()
-	rawdb.WriteTxLookupEntriesByBlock(batch, b)
+	// -1 means skip writing tx lookup entries
+	if bc.cacheConfig.TxLookupLimit != -1 {
```

Comment:
Let's be consistent with using < 0 here

Reply:
I specifically kept it -1 to make it consistent with docs/comments.